### PR TITLE
Feature/fixcliver

### PR DESCRIPTION
**Pull request by gokulavasan**
- Get the CLI version from the build.properties file instead of hardcoding

### CODE REVIEW

File: tigon-client/src/main/java/co/cask/tigon/DistributedMain.java [hunk 1]
```diff
@@ -152,6 +154,15 @@ public void run() {
 
   public void startUp(PrintStream out) throws Exception {
     registerShutDownHook();
+    Properties properties = new Properties();
+    try {
+      InputStream in = getClass().getResourceAsStream("/build.properties");
+      properties.load(in);
+      in.close();
```
Comment (ParthGandhi) [inline tigon-client/src/main/java/co/cask/tigon/DistributedMain.java]: `close()` should happen in a finally block.

File: tigon-client/src/main/java/co/cask/tigon/DistributedMain.java [hunk 2]
```diff
@@ -152,6 +154,15 @@ public void run() {
 
   public void startUp(PrintStream out) throws Exception {
     registerShutDownHook();
+    Properties properties = new Properties();
+    try {
+      InputStream in = getClass().getResourceAsStream("/build.properties");
```
Comment (ParthGandhi) [inline tigon-client/src/main/java/co/cask/tigon/DistributedMain.java]: Do you actually need the "/" at the beginning?

File: tigon-examples/SQLJoinFlow/src/main/java/co/cask/tigon/sqljoinflow/SQLJoinFlow.java [hunk 3]
```diff
@@ -59,7 +59,7 @@ public FlowSpecification configure() {
     @Override
     public void create() {
       setName("SQLinputFlowlet");
-      setDescription("blah blah");
```
Comment (ParthGandhi) [inline tigon-examples/SQLJoinFlow/src/main/java/co/cask/tigon/sqljoinflow/SQLJoinFlow.java:62]: lol

**Comment by ParthGandhi** [comment]
Couple of comments.

File: tigon-client/src/main/java/co/cask/tigon/DistributedMain.java (hunk 1 above)
Comment (ParthGandhi) [inline tigon-client/src/main/java/co/cask/tigon/DistributedMain.java]: Or use: https://code.google.com/p/guava-libraries/wiki/ClosingResourcesExplained

(Assuming we already have a Guava dependency here)

File: tigon-client/src/main/java/co/cask/tigon/DistributedMain.java (hunk 2 above)
Comment (gokulavasan) [inline tigon-client/src/main/java/co/cask/tigon/DistributedMain.java]: Yes it does. Similar to ProjectInfo.java

**Comment by gokulavasan** [comment]
@ParthGandhi Addressed your comments. Take a look.

**Comment by gokulavasan** [comment]
@ParthGandhi Updated PR. Using ProjectInfo instead of constructing new Properties object.

File: tigon-examples/SQLJoinFlow/src/main/java/co/cask/tigon/sqljoinflow/SQLJoinFlow.java [hunk 4]
```diff
@@ -59,7 +59,7 @@ public FlowSpecification configure() {
...
+      setDescription("Performs inner join of two streams <uid, name> & <uid, age> and emits <uid, name, age>");
```
Comment (ParthGandhi) [inline tigon-examples/SQLJoinFlow/src/main/java/co/cask/tigon/sqljoinflow/SQLJoinFlow.java]: "Executes an inner join...".

Also, either use "&" or "and". Not both.

**Comment by ParthGandhi** [comment]
One last comment. Fix and merge.

**Comment by gokulavasan** [comment]
Build passed. Merging.

**Pull request merged**